integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/DistroXClusterStopStartTest.java

```diff
@@ -1,101 +1,61 @@
-    private DistroXTestClient distroXClient;
+public class DistroXClusterStopStartTest extends AbstractMockTest {
 
     @Inject
-    private SdxTestClient sdxTestClient;
+    private DistroXTestClient distroXTestClient;
 
     @Inject
     private DatahubAuditGrpcServiceAssertion auditGrpcServiceAssertion;
 
-    //CB-7294
-    @Ignore
+    @Override
+    protected void setupTest(TestContext testContext) {
+        createDefaultUser(testContext);
+        createDefaultCredential(testContext);
+        createDefaultImageCatalog(testContext);
+        initializeDefaultBlueprints(testContext);
+        createDefaultDatahub(testContext);
+    }
+
     @Test(dataProvider = TEST_CONTEXT_WITH_MOCK)
     @Description(
             given = "there is a running DistroX cluster",
             when = "a scale, start stop called many times",
             then = "the cluster should be available")
     public void testCreateNewRegularDistroXClusterScaleStartStop(MockedTestContext testContext, ITestContext testNgContext) {
         DistroXStartStopTestParameters params = new DistroXStartStopTestParameters(testNgContext.getCurrentXmlTest().getAllParameters());
-        String stack = resourcePropertyProvider().getName();
-        int step = params.getStep();
-        int current = step;
-        DistroXTestDto currentContext = testContext
-                .given(DIX_NET_KEY, DistroXNetworkTestDto.class)
-                .given(DIX_IMG_KEY, DistroXImageTestDto.class)
-                .withImageCatalog()
-                .withImageId(IMAGE_CATALOG_ID)
-                .given(CM_FOR_DISTRO_X, DistroXClouderaManagerTestDto.class)
-                .given(CLUSTER_KEY, DistroXClusterTestDto.class)
-                .withValidateBlueprint(false)
-                .withClouderaManager(CM_FOR_DISTRO_X)
-                .given(stack, DistroXTestDto.class)
-                .withCluster(CLUSTER_KEY)
-                .withName(stack)
-                .withImageSettings(DIX_IMG_KEY)
-                .withNetwork(DIX_NET_KEY)
-                .when(distroXClient.create(), key(stack))
-                .await(STACK_AVAILABLE, key(stack));
-        for (int i = 0; i < params.getTimes(); i++, current += step) {
-            currentContext = currentContext
-                    .when(distroXClient.stop(), key(stack))
-                    .await(STACK_STOPPED, key(stack))
-                    .then(auditGrpcServiceAssertion::stop)
-                    .when(distroXClient.start(), key(stack))
-                    .await(STACK_AVAILABLE, key(stack))
-                    .then(auditGrpcServiceAssertion::start)
-                    .when(distroXClient.scale(params.getHostgroup(), current))
-                    .await(STACK_AVAILABLE, key(stack).withPollingInterval(POLLING_INTERVAL));
-        }
 
-        currentContext
-                .validate();
-    }
+        if (params.getTimes() < 1) {
+            throw new TestFailException("Test should execute at least 1 round of scaling");
```

Comment:
Can we clarify thatthis is a misconfiguration not an execution error?
If I see this error message my instinct would be that the test failed because there was no successful scaling happened.